src/quanteda/financial_metrics.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+
+def generate_financial_metrics(random_returns_df, annual_risk_free=0.00):
+    """
+    Calculate financial metrics based on a DataFrame of random returns on time series.
+
+    Parameters
+    ----------
+    random_returns_df : pd.DataFrame
+        DataFrame containing random return values with a datetime index.
+
+    annual_risk_free : float, default is 0.00.
```

Comment:
to match the Pandas format

```suggestion
    annual_risk_free : float, default 0.00
```